content/01.main_content.md

```diff
@@ -13,4 +25,17 @@ Proxy predictions of ethnicity via name origin are more difficult still; the cho
   For instance, these classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007]. 
   If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of individuals from Spain and Portugal, underrepresentation of Latin American scientists would then go unnoticed in an exclusively computational analysis.
 
-Considering these things, we propose the following recommendations for best practices in auditing scientific systems:
+Recognizing the aforementioned shortcomings, we propose the following recommendations for the creation and deployment of automated auditing tools:
+  1. *Transparency.* Publicly provide all tools, code, and data used in the analysis.
+  This is to enable public scrutiny and transparency to those being audited and those whose data you are using.
+  If the data is private, we recommend providing de-identified or aggregated data. 
+
+  2. *Individuals know best.* Self-identified demographic information should be used in preference to automated predictions.
+  Additionally, use the most granular and representative groups possible.
+
+  3. *Aggregates only.* Audit results should not affect individuals.
+  Gender expression and ethnicity of an individual are not for an algorithm to decide nor are they the focus of an auditing system. 
+  The analyses must focus only on aggregate estimations and any intermediary predictions on an individual should not be used independently.
+
+  4. *Inform the public.* While internal audits can help institutions monitor their practices, the key goal is to remove disparities in public-facing forms of representation. 
+  We believe transparency to the public along with a way to track progress over time is the most effective method to achieve this.
```

Comment:
"transparency to the public" feels awkward to me, but I can't figure out a better way to say it. Maybe just put "transparency"?

Reply:
yes. reworded in https://github.com/greenelab/disparities_opinion/pull/3/commits/099796a991fb6abd0e32cbb137403c9a1a6f9c6b